vendor/engines/secure_rooms/spec/factories/secure_room.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :secure_room, class: SecureRoom do
```

Comment:
I'm not sure the `class` option is necessary. Is there a way we could do `parent: :product` to get these attributes from what already exist?